### DISCUSSION TITLE
Laser cutting cannot cut the complete pattern

### DISCUSSION THREAD

**Discussion by Travelapollo** [arkypita/LaserGRBL#1236] [Q&A]
As shown in the photo, my laser cutting cannot cut the complete pattern
![IMG_0739](https://user-images.githubusercontent.com/76719925/103259453-70d78900-49d4-11eb-84ef-df97bf4c8011.JPG)
![IMG_0738](https://user-images.githubusercontent.com/76719925/103259445-661cf400-49d4-11eb-9bde-21eafddeba2f.JPG)

grbl configuration is as follows
$0=10 (Step pulse time)
$1=255 (Step idle delay)
$2=0 (Step pulse invert)
$3=11 (Step direction invert)
$4=0 (Invert step enable pin)
$5=1 (Invert limit pins)
$6=0 (Invert probe pin)
$10=255 (Status report options)
$11=0.010 (Junction deviation)
$12=0.002 (Arc tolerance)
$13=0 (Report in inches)
$20=0 (Soft limits enable)
$21=1 (Hard limits enable)
$22=1 (Homing cycle enable)
$23=111 (Homing direction invert)
$24=2000.000 (Homing locate feed rate)
$25=2000.000 (Homing search seek rate)
$26=250 (Homing switch debounce delay)
$27=5.000 (Homing switch pull-off distance)
$30=1000 (Maximum spindle speed)
$31=0 (Minimum spindle speed)
$32=1 (Laser-mode enable)
$100=80.000 (X-axis travel resolution)
$101=80.000 (Y-axis travel resolution)
$102=400.000 (Z-axis travel resolution)
$110=500.000 (X-axis maximum rate)
$111=500.000 (Y-axis maximum rate)
$112=200.000 (Z-axis maximum rate)
$120=20.000 (X-axis acceleration)
$121=20.000 (Y-axis acceleration)
$122=50.000 (Z-axis acceleration)
$130=400.000 (X-axis maximum travel)
$131=400.000 (Y-axis maximum travel)
$132=250.000 (Z-axis maximum travel)

and my Border Speed is set 80mm/min
but the same situation still appears,
how can I set or adjust the parameters?

**Comment by arkypita** [comment]
Your laser looks very out of focus.
If you can't get a thinner spot, you'll hardly be able to cut anything.

**Reply by Travelapollo** [reply]
I hope there is a function that can delay the movement, which means that when the laser is turned on, it can move and cut after a few seconds.

**Reply by Travelapollo** [reply]
as #189

**Reply by arkypita** [reply]
> I hope there is a function that can delay the movement, which means that when the laser is turned on, it can move and cut after a few seconds.

Yes, this command exists but I never implement this feature because a well designed hardware does not need a delay to cut.
What do you think about your problem: is the laser that turn on in delay respect move, or it turn on "in time" synchronized with start move, but the material does not burn at first fire?
